README.md

```diff
@@ -1,96 +1,179 @@
+at https://azureiotsuite.com.
+
+>  This solution is currently in preview. The privew offers many features
+but there are known bugs. You can refer to our [Issues List](issues) in
+this repo (and in any submodules) to see known requests and issues. Feel
+free to submit new issues (and submit PRs for fixes if you'd like).
+
+Getting Started
+===============
+
+## Prerequisites
+* Azure Subscription ([permissions guidelines](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-permissions))
+
+## Setup
+1. Check out the repository and submodules
+    ```
+    git clone --recursive https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet.git
+    ```
+1. Set up command line interface for deployments
+    ```
+    cd pcs-cli
+    npm install
+    npm start
+    npm link
+    ```
+1. From `pcs-cli`, sign into your account for pcs-cli with `pcs login`
+
+## Standard Deployment
+```
+pcs -s standard --runtime java
+```
+
+## Basic Deployment
+```
+pcs --runtime java
+```
+
+## Local Deployment
+To set up a Remote Monitoring Solution locally see
+[Running all PCS microservices locally](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#running-all-pcs-microservices-locally)
+
+Common Scenarios
+================
+
+## All done? Connect a device!
+By default the solution uses simulated devices. You can start adding your
+own devices with the instructions here [Connect a physical device](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#connect-a-physical-device)
+
+Standard vs. Basic Deployments
+==============================
+### Standard
+The standard deployment is a production-ready deployment a developer can
+customize and extend to meet their needs. For reliability and scale, application
+microservices are built as Docker containers and deployed using an orchestrator
+([Kubernetes](https://kubernetes.io/) is default). The orchestrator is
+responsible for deployment, scaling, and management of the application.
+
+Creating a Standard solution will result in the following Azure services being
+provisioned into your Azure subscription at cost: 
+* 1 Azure Active Directory application
+* 1 Azure Container Service 
+* 3 Virtual Machines (Standard D1 V2 (1 core, 3.5GB memory))
+* 1 IoT Hub (S1 – Basic tier)
+* 1 Cosmos DB account (Standard)
+* 1 Storage account (Standard -GRS)
+* 1 Web Application
+
+### Basic
+Basic deployment is geared toward showcasing the solution. To reduce the cost
+of this demonstration, all of the microservices are deployed in a single
+virtual machine; this is not considered a production-ready architecture.
+ 
+Our Standard deployment option should be used when you are ready to customize
+a production-ready architecture, built for scale and extensibility.
+ 
+Creating a Basic solution will result in the following Azure services being
+provisioned into your Azure subscription at cost: 
+* 1 Azure Active Directory application
```

Comment:
shoudl we provide a brief descritpion of why the resources are provisioned?  e.g. "The VM hosts your microservice, enabling simulation, generating alarms based on telemetry, hosting your UI, etc."